src/logic/safe/store/reducer/allTransactions.ts

```diff
@@ -28,16 +22,13 @@ export default handleActions(
       },
     ) => {
       const { safeAddress, transactions } = action.payload
-      const oldTxs = (state.transactions && state.transactions[safeAddress]) || []
+      const oldTxs = (state && state[safeAddress]) || []
```

Comment:
we should never have a case where state is undefined, so check for state is not needed